default/stringtables/en.txt

```diff
@@ -15434,6 +15434,9 @@ Protection Focus
 GASEOUS_LABEL
 Gas Giant Environment
 
+MINDLESS_LABEL
```

Comment:
This is the entry for showing the effect. Stringtable entries for species description in encyclopedia MINDLESS_SPECIES and MINDLESS_SPECIES_DESC are missing (similar to GASEOUS and GASEOUS_DESC entries)